base_user_role/wizards/group_groups_into_role.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Sodexis
+# License OPL-1 (See LICENSE file for full copyright and licensing details).
+
+
+from odoo import fields, models
+
+
+class GroupGroupsIntoRole(models.TransientModel):
+    """
+    This wizard is used to group different groups into a role.
+    """
+
+    _name = "group.groups.into.role"
+    _description = "Group groups into a role"
+    name = fields.Char(
+        required=True,
+        help="Group groups into a role and specify a name for this role",
+    )
+
+    def create_role(self):
+        selected_group_ids = self.env.context.get("selected_group_ids", [])
```

Comment:
```suggestion
        selected_group_ids = self.env.context.get("active_ids", [])
```

should work, don't you think ? (and avoid to add extra context in the action).